src/app/route.scss

```diff
@@ -0,0 +1,108 @@
+.route-card {
+  width: 50%;
+  margin-top: 3%;
+  margin-left: 25%;
+  height: 80%;
+  padding: 0;
+  background-color: #5f9ea099;
+  color: white;
+  h1 {
+    text-align: center;
+    font-weight: bold;
+    font-size: 3rem;
+    margin: 0;
+  }
+  em {
+    font-size: 0.9rem;
+  }
+}
+
+.text-page-route {
+  text-align: center;
+  font-size: 1.1rem;
+}
+
+.tech-page-route {
+  text-align: center;
+  font-size: 0.9rem;
+  font-style: italic;
+  font-weight: bold;
+  color: rgba(0, 255, 149, 0.859);
+  text-shadow: rgb(0, 149, 255) 2px 2px 2px;
+}
+
+.publ-text {
+  font-size: 1rem;
+}
+
+.contact-icon {
+  font-size: xx-large;
+  margin-right: 2%;
+  vertical-align: middle;
+  margin-bottom: 10px;
+  color: black;
+}
+
+.nav-links {
+  color: blue;
+  text-decoration: underline;
+  &:hover {
+    color: white;
+    text-decoration: underline;
+  }
+}
+
+@mixin logo-route($prop1, $prop2, $prop3) {
+  font-size: 5rem;
+  color: $prop1;
+  background-color: $prop2;
+  margin-right: 2%;
+  margin-top: $prop3;
+}
+
+.JS-logo-route {
+  @include logo-route(#f7df1e, black, 0);
+}
+
+.HTML-logo-route {
+  @include logo-route(white, #e54c21, 0);
+}
+
+.CSS-logo-route {
+  @include logo-route(white, #004ce8, 0);
+}
+
+.React-logo-route {
+  @include logo-route(#01dafe, #222222, 0);
+}
+
+.TS-logo-route {
+  @include logo-route(#007bcd, white, 4%);
+}
+
+.Sass-logo-route {
+  @include logo-route(#ce679a, white, 4%);
+}
+
+.Git-logo-route {
+  @include logo-route(#f05030, white, 4%);
+}
+
+.Next-logo-route {
+  @include logo-route(black, white, 4%);
+}
+
+%extend-logo-svg-route {
```

Comment:
I'm not sure why we need this? 
Also the name does not tell me semantically what does it do/how it is intended to be used.

One idea - remove the classes extending this and just make a normal class from this one with appropriate name.